x-pack/plugins/ml/server/routes/schemas/job_service_schema.ts

```diff
@@ -42,6 +42,7 @@ export const forceStartDatafeedSchema = schema.object({
 export const jobIdsSchema = schema.object({
   /** Optional list of job IDs. */
   jobIds: schema.maybe(schema.arrayOf(schema.maybe(schema.string()))),
+  excludeGenerated: schema.maybe(schema.boolean()),
```

Comment:
this should also be added to the datafeeds schema

Reply:
I removed this here as it's no longer needed 
 https://github.com/elastic/kibana/pull/88898/commits/e9ad8f16c6e349a3d7bbc5199ad6d12de0f1bb72